py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -143,6 +143,10 @@ class RemoteConnection:
     )
     _ca_certs = os.getenv("REQUESTS_CA_BUNDLE") if "REQUESTS_CA_BUNDLE" in os.environ else certifi.where()
 
+    # Class variables for headers
+    extra_headers = None
+    user_agent = f"selenium/{__version__} (python {platform.system().lower()})"
```

Comment:
Should https://github.com/SeleniumHQ/selenium/blob/11c4a2c487cbc704ea47a9c392382b7697586cb4/py/selenium/webdriver/remote/remote_connection.py#L210 refer to this user_agent?

Reply:
Yes, I will update this, although the corresponding test needs to be changed. Currently, we were making and Assertion for `mac)` in OS list but my change makes it `mac`, so for now i have added both combination in the tests.